include/entity.h

```diff
@@ -189,6 +189,14 @@ typedef struct {
     /* 0x9C */ s16 unk9C;
 } ET_Weapon29;
 
+typedef struct {
+    s16 timer;
+    s16 unk7E;
+    s32 unk80;
+    s16 pad[21];
+    s16 foodId;
+} ET_Food;
```

Comment:
I was wondering if you could re-use `ETWeapon`? `lifetime` and `timer` means the same thing. While `foodId` seem to match with `equipId`, which also mean the same thing.

Reply:
The fatal mismatch is that food has `s32 unk80` while Weapon has `s16 unk80` and `s16 unk82`.

Reply:
argh okay, that's fine then